### ISSUE TITLE
Read file from hdfs to memory and write file from memory to hdfs.

### ISSUE THREAD

**Issue by taleslimaf** [mtth/hdfs#163]
I would like to know if it is possible to read the file from hdfs to memory and write file from memory to hdfs.

**Comment by alex-bo** [comment]
Yes, it is possible. Check out class hdfs.client.Client methods read() and write() in the documentation:
- https://hdfscli.readthedocs.io/en/latest/api.html#hdfs.client.Client.read
- https://hdfscli.readthedocs.io/en/latest/api.html#hdfs.client.Client.write

**Comment by taleslimaf** [comment]
Sorry, just to complement, it would be something in Bytes.

**Comment by mtth** [comment]
Late to the party - both functions above will deal with bytes if you don't specify an `encoding`.
